application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -331,7 +334,33 @@ public function checkRequiredStages(Validation $validation, $completed_stages, $
 	 * @param  Array      $attributes
 	 * @param  Array      $fullData
 	 */
-	public function checkRequiredAttributes(Validation $validation, $attributes, $fullData)
+	public function checkRequiredPostAttributes(Validation $validation, $attributes, $fullData)
+	{
```

Comment:
I think we're still missing attributes though, what about required attributes not on required stages?